estimagic/dashboard/server_functions.py

```diff
@@ -0,0 +1,82 @@
+"""Functions for setting up and running the BokehServer displaying the dashboard."""
+import asyncio
+from functools import partial
+
+from bokeh.application import Application
+from bokeh.application.handlers.function import FunctionHandler
+from bokeh.command.util import report_server_init_errors
+from bokeh.server.server import Server
+
+from estimagic.dashboard.dashboard import run_dashboard
+
+
+def run_server(queue, notebook, port):
+    """
+    Setup and run a server creating und continuously updating a dashboard.
+
+    The main building of the dashboard is done in run_dashboard.
+    Here this function is only turned into an Application and run in a server.
+
+    Args:
+        queue (Queue):
+            queue to which originally the parameters DataFrame is supplied and
+            to which later the updated parameter Series will be supplied.
+
+        notebook (bool):
+            whether the function is called in a jupyter notebook.
+
+        port (int):
+            port at which to display the dashboard.
+
+    """
+    asyncio.set_event_loop(asyncio.new_event_loop())
+
+    apps = {"/": Application(FunctionHandler(partial(run_dashboard, queue=queue)))}
+
+    server = _setup_server(apps, notebook, port)
+
+    server._loop.start()
+    server.start()
+    if notebook is True:
```

Comment:
- The dashboard works from a notebook with notebook=False and notebook=True, but the lines in the convergence plot have different color scales. Any Idea why this is the case? Do we still need the notebook option? If we still need it, could we use something like [this](https://stackoverflow.com/a/24937408) instead?

Reply:
You are right. I removed the notebook option everywhere.